### DISCUSSION TITLE
no TSA forecast data

### DISCUSSION THREAD

**Discussion by jamwxman** [weather-gov/api#389] [Q&A]
https://api.weather.gov/points/35.5,-95
 gives
 https://api.weather.gov/gridpoints/TSA/76,74/forecast
 gives
 
 {
    "correlationId": "3b59d1fd-7a9f-4605-88ab-5a075da5ce44",
    "title": "Data Unavailable For Requested Point",
    "type": "https://api.weather.gov/problems/InvalidPoint",
    "status": 404,
    "detail": "Unable to provide data for requested point 90,120.5517605",
    "instance": "https://api.weather.gov/requests/3b59d1fd-7a9f-4605-88ab-5a075da5ce44"
}

It appears no forecast data is returned for points in TSAs area and occasionally this happens for other points in other CWAs.

**Comment by joepperkins** [comment]
Try truncating the precision to 4 places right of the decimal point example xx.xxxx.

I have not seen anywhere it will accept precision more than 4 decimal places.

Joe Perkins

On Oct 23, 2019 10:01 AM, jamwxman <notifications@github.com> wrote:

https://api.weather.gov/points/35.5,-95
gives
https://api.weather.gov/gridpoints/TSA/76,74/forecast
gives

{
"correlationId": "3b59d1fd-7a9f-4605-88ab-5a075da5ce44",
"title": "Data Unavailable For Requested Point",
"type": "https://api.weather.gov/problems/InvalidPoint",
"status": 404,
"detail": "Unable to provide data for requested point 90,120.5517605",
"instance": "https://api.weather.gov/requests/3b59d1fd-7a9f-4605-88ab-5a075da5ce44"
}

It appears no forecast data is returned for points in TSAs area and occasionally this happens for other points in other CWAs.

**Comment by StephenClouse** [comment]
This is a known issue on our end. We've requested a restart of the backend service to correct it.
